package.json

```diff
@@ -85,6 +93,7 @@
     "parcel-bundler": "^1.12.4",
     "prettier": "^1.19.1",
     "pretty-quick": "^2.0.1",
+    "regenerator-runtime": "^0.13.5",
```

Comment:
Hmm, vad använder du "regenerator-runtime" för?

Reply:
Om jag förstod det rätt så behövdes den för att få ES6 att funka i testfallen (i vissa fall, som vanligt luddigt).

Se: https://jestjs.io/docs/en/22.x/getting-started.html (sök på regenerator-runtime). Input är välkommen!

Reply:
Okej, du har rätt. Man ska installera det. Även om jag hittills aldrig installerade det och Jest+Babel fungerar bra... 😉 

Vi använder väl NPM 6 - "Explicitly installing regenerator-runtime is not needed if you use npm 3 or 4 or Yarn" (?) 😕

Reply:
Ja, jag använder NPM 6 och det funkade inte ... vette tusan